### PR TITLE
refactor: Copilot view settings setter/getter

### PR DESCRIPTION
**Pull request by jfcherng**
Waiting for other PRs to be merged...

---

And maybe we want to further do 
```diff
-COPILOT_VIEW_SETTINGS_PREFIX = "copilot.completion"
+COPILOT_VIEW_SETTINGS_PREFIX = "copilot"
```
I am not sure.

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -2,6 +2,21 @@
 import textwrap
 
 import sublime
+from LSP.plugin.core.typing import Any, Optional
+
+from .constants import COPILOT_VIEW_SETTINGS_PREFIX
+
+
+def get_copilot_view_setting(view: sublime.View, key: str, default: Optional[Any] = None) -> Any:
```
Comment (timfjord) [inline utils.py:10]: I don't think that `Optiona[Any]` makes sense, as `Any` already covers `None`

Comment (jfcherng) [reply utils.py:10]: Technically they are not the same. `Any` completely drops type checking so you can do something like `default.append(x)`. But you can't do that for `Optional[Any]` case.

Comment (jfcherng) [reply utils.py]: done in [80b1d38](https://github.com/TheSecEng/LSP-copilot/pull/9/commits/80b1d38887520c68e3a70f5fe9d1a66342179ff1)

**Comment by jfcherng** [comment]
I think this PR has been ready for review.

**Review by TheSecEng — APPROVED** [review]

**Pull request merged**